string task_name
topological_patroller/CheckPointTask[] patrol
